### ISSUE TITLE
2024 restorecord source back up - scammers exposed again

### ISSUE THREAD

**Issue by rc-mill** [authgg/restorecord-new-source-code#6]
restorecord 2024 source code back up, xenos1337 the scammer and selling user data https://nelsoncybersecurity.com/xenos-sell-data.mp4

WATCH FOR PROOF https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4

@ripkova @MinisterPP @jacobsomer @nakinami @azurlul @8FrMc @mesh2l @InstinctTheDevil @Gregory-x @INCOD3 @pubgseller @goodluck4you @MernGG @l29 @Duchy12 @Fweak @Blomgreen @xKiian @Zxxka @shiesty-kid @NobiteK @DuHastNHLatte @furaar @paucofficial @Gamer-Dave @KTDR @cumsoft @MIKHAIL-4T4 @ipas0 @LeoonThePaster @GotYouLittleBoy @OMUTU0001 @Proaids26 @longxinyun-security @wonderzdh @Grassless1337 @UCFoxi @AgentFN @3r4y @wokonly @Th3R3ality @spacerp123 @itagragonpro8551 @Jankovik @AuSLove0001 @Traumatism @lyf-is-coding @kajtus32 @Im2rnado @Darkdog123

**Comment by rc-mill** [comment]
restorecord 2024 source code back up, xenos1337 the scammer and selling user data https://nelsoncybersecurity.com/xenos-sell-data.mp4

WATCH FOR PROOF https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4

@jonassyon @psycal @xkp95175333 @yakitori5442 @OrchaniousS @05m @HimikoValk @Eastonn @f0m @2636166082 @tsuku1337 @sp11xy@sin-tex @0iy @4u4play @lksload @DevilZ46 @DIlIlII @Angelbooy @Nugzy69 @rikodot @dopuke @H4X4U @Bvaudin @422i @XeraFiu-YTB @UnrealSkill-VIP @HotshotGTV @Zedvy @xxezn @jamescharlie9 @finefn @SonnyLikesCode @ILovePastes @ohkriss @ExpLife0011 @Jackth10 @onlinekid @Skxxet @Avapt @Edwin32172 @decries @Heashey-zz @un4ckn0wl3z @vatsalmasrani @masterlinnn @jev0n @SsValor @ALEHACKsp @JBris @remotemachine1 @JBris

**Comment by rc-mill** [comment]
restorecord 2024 source code back up, xenos1337 the scammer and selling user data https://nelsoncybersecurity.com/xenos-sell-data.mp4

WATCH FOR PROOF https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4

@fengjixuchui @SleekZ @rafalohaki @DarleneC @kifast @cn6u9 @ScalletaZ @vlaqy @Fuwn @Hymerine @wnelson03

**Comment by authgg** [comment]
Yep that's true, GitHub **confirmed 100%** that this source code is real & made by the actual xenos guy -> See this link https://github.com/authgg/restorecord-new-source-code/commit/259b638ab6f06ab0a5c7404b1b37fc27d45e727a

![image](https://github.com/authgg/restorecord-new-source-code/assets/67508850/1c892072-eb1b-429f-ada0-4d792b9580f7)

**Comment by xKiian** [comment]
Shut the fuck up. Why is you mass pinging

**Comment by Traumatism** [comment]
Do not mention me on that tiny little buddy

**Comment by Blomgreen** [comment]
Move on with your life Mak

**Comment by rafalohaki** [comment]
sure

**Comment by NobiteK** [comment]
why im mentioned here bruh

**Comment by rc-mill** [comment]
@xKiian unfollow the banned scammer https://cracked.io/xenos1337 if you don't want to be notified about his wrongdoings

**Comment by rc-mill** [comment]
@Traumatism @Blomgreen @NobiteK unfollow the banned scammer https://cracked.io/xenos1337 if you don't want to be notified about his wrongdoings

**Comment by xKiian** [comment]
Bro is in a film

**Comment by Traumatism** [comment]
> @Traumatism @Blomgreen @NobiteK unfollow the banned scammer https://cracked.io/xenos1337 if you don't want to be notified about his wrongdoings

I don't give a f leave me alone

**Comment by rc-mill** [comment]
@Traumatism then unfollow him, but if you keep supporting a scammer why should I care how you feel? https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4

**Comment by Traumatism** [comment]
> @Traumatism then unfollow him, but if you keep supporting a scammer why should I care how you feel? https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4

Unfollowed him so you leave me alone now go away zoro

**Comment by Blomgreen** [comment]
> @Traumatism @Blomgreen @NobiteK unfollow the banned scammer https://cracked.io/xenos1337 if you don't want to be notified about his wrongdoings

or I could unsubscribe from the email. Also, nice deleting comments.
Move on with your life, Mak. You're pathetic

**Comment by Blomgreen** [comment]
![Screenshot 2024-07-05 221829](https://github.com/authgg/restorecord-new-source-code/assets/90788472/482b67ed-398f-4178-801a-e07124977f4f)
Yawn lil bro. Go out and party instead of being an online Asperger

**Comment by NobiteK** [comment]
bro thinks he's main character 💀

**Comment by rc-mill** [comment]
@Blomgreen @NobiteK supporting scamming & selling personal information is sad. also wow so edgy making fun of disabled people 🙄 real original and clever there 💀 

https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4
https://nelsoncybersecurity.com/xenos-sell-data.mp4

**Comment by Blomgreen** [comment]
Thinking you're weird doesn't constitute supporting a scammer. booh, you're boring. Move on with your life. It's been years since you handed over restorecord

**Comment by NobiteK** [comment]
u is sherlock holmes?

**Comment by rc-mill** [comment]
@Blomgreen @NobiteK yeah nah they're banned for scamming, you must be a scammer too https://cracked.io/xenos1337 and https://cracked.io/bl4ckbl1zz and https://cracked.io/zebratic

Youtuber with 800K subs exposed restorecord also, it's almost like they're doing something illegal & wrong 🤯 https://www.youtube.com/watch?v=d0h4QPqAwss

**Comment by Blomgreen** [comment]
Are you slandering me right now?

**Comment by NobiteK** [comment]
ok men who cares

**Comment by rc-mill** [comment]
@Blomgreen cry more, supporting scamming goofy ass

**Comment by rc-mill** [comment]
@NobiteK ah look at this skid. he hardcodes the copyright date & tries to PROTECT HTML CODE 🤣 🤣 don't let bro cook ever again..

![image](https://github.com/authgg/restorecord-new-source-code/assets/162258149/fb4d870a-a949-4ca3-8095-52caf6030c07)

**Comment by Blomgreen** [comment]
It's 4:45pm in Florida lil bro. Go hit the gym or something

**Comment by NobiteK** [comment]
bro what are you yapping about

**Comment by authgg** [comment]
Guys stay respectful or I'm going to close this issue

**Comment by rc-mill** [comment]
> bro what are you yapping about

you would know if you had a clue about programming and weren't a **trashcan skid** that contributes nothing

**Comment by NobiteK** [comment]
bro how old are you?

**Comment by Blomgreen** [comment]
> Guys stay respectful or I'm going to close this issue

Bro talking like it's not the same person on both accounts. Stop Larping Mak.
have you been outside today? Have you done your push-ups for the day?

**Comment by Blomgreen** [comment]
> Lol y'all are children. And nah rc-mil is the restorewhore guy, not me bro

ON A 3RD ACCOUNT NOW, THIS IS WIILD

**Comment by Blomgreen** [comment]
Deleting my messages like crazy for calling him out for using 3 accounts that's wild.
Why am I interacting with such a weirdo I gotta go do sumn. You probably can't follow that thought tho

**Comment by rc-mill** [comment]
bro will randomly call me alt but won't believe proof against his daddy xenos 🤣 🤡 

https://cracked.io/xenos1337
https://cracked.io/bl4ckbl1zz
https://cracked.io/zebratic

https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4
https://nelsoncybersecurity.com/xenos-sell-data.mp4

**Comment by NobiteK** [comment]
go outside

**Comment by rc-mill** [comment]
@NobiteK stop scamming skid

**Comment by NobiteK** [comment]
nice iq

**Comment by Blomgreen** [comment]
Weird your messages aren't being removed.. Maybe because it's the same guy.
Go outside, William.

And stop slandering the random people you just added, that would be a crime.

**Comment by rc-mill** [comment]
> nice iq

yeah your iq is room temp the proof is in front of you and you're like "but scamming good"

https://pub-d1677a93e4284baf8c2d931a541f3fd6.r2.dev/restorecord-2024-source-leak-data-breach-fraud-scams.mp4
https://nelsoncybersecurity.com/xenos-sell-data.mp4

**Comment by Blomgreen** [comment]
Slander is a crime in Florida. Go do your pushups lil bro

**Comment by NobiteK** [comment]
🤓

**Comment by 1neez** [comment]
ur a larp lil bro

**Comment by rc-mill** [comment]
@Blomgreen Julius the scammer is pressed, go use fedded Protonmail you ignorant skid

**Comment by Blomgreen** [comment]
![Screenshot 2024-07-05 225936](https://github.com/authgg/restorecord-new-source-code/assets/90788472/ecaa787b-73dd-4f6d-8800-74849e0e16b0)

**Comment by rc-mill** [comment]
@1neez Ok Julius alternate, nobody mentioned you in here. obvious alt 🤡

**Comment by 583d** [comment]
bro what ur a faxxeet

**Comment by rc-mill** [comment]
> ![Screenshot 2024-07-05 225936](https://private-user-images.githubusercontent.com/90788472/346224618-ecaa787b-73dd-4f6d-8800-74849e0e16b0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjAyMTM0ODYsIm5iZiI6MTcyMDIxMzE4NiwicGF0aCI6Ii85MDc4ODQ3Mi8zNDYyMjQ2MTgtZWNhYTc4N2ItNzNkZC00ZjZkLTg4MDAtNzQ4NDllMGUxNmIwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA1VDIwNTk0NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk3NWJkMGYxZWQ3MDIxYzhkYjI2ZmNiODUyZjVkZGEyMDdhYWE5MzJmNmRmNjBlZmI3NTE3ZWQ0NDE1YjI3ZjYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Pg33QJBfZh4SouFy85fvLHIbNfuTgAzEO_ummmVL-pw)

Julius is seething, good thing I'm not in that gay hating state like you are. You bigot scammer

**Comment by rc-mill** [comment]
> bro what ur a faxxeet

skid is renaming himself as we speak 💀 💀 Julius your mommy said go do your homework headass

**Comment by Blomgreen** [comment]
No University of South Florida?

**Comment by rc-mill** [comment]
i would never live in florida they hate gay people which you probably like because you're a hateful harasser who also enjoys scamming

**Comment by Blomgreen** [comment]
Go do some pushups

**Comment by rc-mill** [comment]
> Go do some pushups

Julius you can't even lift your own bullshit, pipe down there skiddie

**Comment by 583d** [comment]
shux up skid

**Comment by 583d** [comment]
get a job

**Comment by rc-mill** [comment]
@583d ok Julius on an alt, whatever you say
